scripts/30mini-iso-menu

```diff
@@ -32,12 +32,12 @@ iso_chooser_step1() {
         tty2 linux
     chvt 1
 
-    if [ ! -f /out.vars ] ; then
+    if [ ! -f /mini-iso-menu.vars ] ; then
         echo "ISO menu failed, debug shell"
         /bin/sh
```

Comment:
It's 100% not the topic of this PR but is the "panic" function from initramfs-tools more appropriate here?

Reply:
yea, I'm inconsistent on the initramfs integration, that's worth looking into.  Future PR.